src/provenance/main.py

```diff
@@ -117,19 +117,27 @@ def get_dependencies(drv_path: str, recursive: bool = False) -> list[dict]:
     return dependencies
 
 
-def get_external_parameters(drv_path: str, metadata: BuildMeta) -> dict:
-    """Get externalParameters from env variable and add derivation"""
+def get_external_parameters(metadata: BuildMeta) -> dict:
+    """Get externalParameters from env variable"""
+
     params = json.loads(metadata.external_parameters or "{}")
 
-    # add derivation path always to params
-    params["derivation"] = drv_path
```

Comment:
drv_path is then not used even as the default, in case environment does not set the path. Is this the wanted behavior?

Reply:
yes because you could have a build type document that specifies no external parameters are supplied